.github/workflows/dependency_graph.yml

```diff
@@ -12,6 +12,21 @@ jobs:
         runs-on: ubuntu-latest
         steps:
             -   uses: actions/checkout@v2
+
+            -   name : Configure GitHub and create branch
+                env:
+                    GITHUB_TOKEN: ${{ secrets.BALLERINA_BOT_TOKEN }}
+                run: |
+                    git config user.name ${{ secrets.BALLERINA_BOT_USERNAME }}
+                    git config user.email ${{ secrets.BALLERINA_BOT_EMAIL }}
+                    echo "BRANCH_EXISTS=0" >> $GITHUB_ENV
```

Comment:
Can't we directly set this as an environment variable similar to line no. 18?

Reply:
The value is being updated dynamically.

Reply:
Still, we need to define it first right? This line is just defining the `BRANCH_EXISTS` variable with the initial value of `0`.

Reply:
Updated